README.md

```diff
@@ -112,6 +112,18 @@ Plug 'easymotion/vim-easymotion', Cond(!exists('g:vscode'))
 Plug 'asvetliakov/vim-easymotion', Cond(exists('g:vscode'), { 'as': 'vsc-easymotion' })
 ```
 
+If you do not want to define `Cond` function, here is how you do it:
```

Comment:
Not really sure we need to mention `Cond` at all. We should probably remove it from the docs and code entirely. It's noisy to mention multiple ways to do something in the docs, for no reason.

Reply:
Yeah, I agree with you. How about cleaning the whole section like this? I've updated my commit. Please have a look.